### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by harimkang**
### Summary

- Update Mobile Environment
Before:
image
After:
image

- Update Hugging-Face pretrained model tutorials
- Update CHANGELOGS

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have ran e2e tests and there is no issues.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: docs/source/guide/tutorials/advanced/huggingface_model.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Semantic Segmentation
+~~~~~~~~~~~~~~~~~~~~~
+
+- **Pixel-wise Labeling**: Prepare datasets with pixel-wise labeling for semantic segmentation and use them to train your models.
+
+OpenVINO IR Model Conversion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **Model Optimization and Conversion**: Convert the trained models into the IR format using the OpenVINO toolkit to optimize performance across various hardware platforms.
+- **Deployment Ready**: The converted models are ready for immediate deployment, and OpenVINO™ Training Extensions simplifies this process, supporting users in easily deploying models to edge devices or servers.
```
Comment (eunwoosh) [inline docs/source/guide/tutorials/advanced/huggingface_model.rst]: It looks generalized information of each task. Maybe this part isn't necessary. If you think it's necessary, please resolve it :)

**Review by eunwoosh — DISMISSED** [review]
LGTM. I have one minor comments. Please take a look :)

File: docs/source/guide/tutorials/advanced/huggingface_model.rst (hunk 1 above)
Comment (harimkang) [reply docs/source/guide/tutorials/advanced/huggingface_model.rst]: i agree, thanks!
Done - https://github.com/openvinotoolkit/training_extensions/pull/3768/commits/085dcc695d48b6327236085dea4c7c58d6197d0d

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,11 @@ All notable changes to this project will be documented in this file.
 
 - Add RT-DETR model for object detection task
   (https://github.com/openvinotoolkit/training_extensions/pull/3741)
+- Add Multi-Label & H-label Classification with torchvision models by @harimkang (https://github.com/openvinotoolkit/training_extensions/pull/3697)
```
Comment (sungchul1) [inline CHANGELOG.md]: Referring to previous versions' logs, an author for PR is omitted.

Comment (harimkang) [reply CHANGELOG.md]: Thanks, updated it - https://github.com/openvinotoolkit/training_extensions/pull/3768/commits/cba2f781114e5e780058b4da0582bc0d8bd68c4b

**Review by eunwoosh — DISMISSED** [review]

**Review by sungchul1 — APPROVED** [review]

**Review by eunwoosh — APPROVED** [review]

**Pull request merged**